COMPATIBILITY.md

```diff
@@ -16,3 +17,11 @@ The below matrix shows the compatibility of the `opensearch-net` with versions o
 | 1.2.4 | 1.0.0 |
 | 1.3.0 | 1.0.0 |
 | 1.3.1 | 1.0.0 |
+| 1.3.2 | 1.0.0 |
+| 1.3.3 | 1.0.0 |
+| 2.0.0 | 1.0.0 |
+| 2.0.1 | 1.0.0 |
```

Comment:
I think we should update the compatibility matrix to say that client 1.0 is compatible with OpenSearch 1.x, and client 1.1 is compatible with OpenSearch 1.x and 2.x.

Reply:
Ggree, if 1.x means literally 1.x. I asked in [inital PR](https://github.com/opensearch-project/opensearch-net/pull/43#discussion_r879904991) to use 1.x/2.x instead of listing out all versions, it's easier to read.

The next client version should be 1.1 since there are no breaking changes, so maybe this? Users are likely to be already using some version of OpenSearch. They can lookup what are the compatible client versions, instead of the other way around.

```
| OpenSearch Version | Client Version |
|--------------------|----------------|
| 1.x                | 1.0.0, 1.1.0   |
| 2.x                | 1.1.0          |
```

Reply:
You could reverse the columns making it easier to grok ;)

Reply:
Thanks, updated in 0d58b5b71.